site/src/pages/release-notes/1.15.0.mdx

```diff
@@ -0,0 +1,351 @@
+---
+date: 2022-03-24
+---
+
+## 🌸 Highlights
+
+- gRPC health check protocol
+- Multipart file uploads in annotated services
+- Preliminary GraphQL support in <type://DocService> 
+- Refactored DNS resolver cache with higher hit ratio and reduced traffic
+- New metrics that help you get alerts before your TLS certificates are expired
+
+## 🌟 New features
+
+- <type://Server> now registers two gauges about TLS certificate expiry, so you can get alerts before
+  the certificates are expired. #4075 #4082
+  - `armeria_server_certificate_validity`
+    - `1` if certificate is in validity period. `0` otherwise.
+  - `armeria_server_certificate_validity_days`
+    - The number of days before certificate expires, or `-1` if expired.
+- You can now specify a `File` or `Path` parameter in your annotated service method to handle multipart file
+  uploads. #3578 #4061
+  ```java
+  Server
+    .builder()
+    .annotatedService(new Object() {
+      @Post("/upload")
+      @Consumes("multipart/form-data")
+      String upload(@Param File file) {
+        return "Successfully uploaded a file to " + file;
+      }
+    })
+    .build();
+  ```
+- You can now use the `||` (logical OR) operator in parameter and header predicate expressions. #4116 #4138
+  ```java
+  // "Hello!" for /greet or /greet?casual=false
+  // "Hi!" for /greet?casual or /greet?casual=true
+  Server
+    .builder()
+    .route()
+      .get("/greet")
+      .matchesParam("casual || casual=true") // 👈👈👈
+      .build((ctx, req) -> HttpResponse.of("Hi!"))
+    .route()
+      .get("/greet")
+      .matchesParam("!casual || casual!=true") // 👈👈👈
+      .build(ctx, req) -> HttpResponse.of("Hello!"))  
+  ```
+  <Tip>
+  
+  You can use the same syntax in [annotated services](../docs/server-annotated-service)
+  using <type://@MatchesParam> and <type://@MatchesHeader>.
+  
+  </Tip>
+- You can now enable [gRPC health check protocol](https://github.com/grpc/grpc/blob/master/src/proto/grpc/health/v1/health.proto)
+  with <type://GrpcHealthCheckService> #3146 #3963
+  ```java
+  Server
+    .builder()
+    .service(
+      GrpcService
+        .builder()
+        .addService(GrpcHealthCheckService.of()) // 👈👈👈
```

Comment:
Should it be better to show `enableHealthCheckService(boolean)` as a sensible default option?

Reply:
Good point. Updated